File: main/src/ca/uwaterloo/flix/runtime/solver/api/predicate/TruePredicate.scala
package ca.uwaterloo.flix.runtime.solver.api.predicate

/**
  * Represents the true predicate.
  */
class TruePredicate extends Predicate {
  /**
    * Returns a string representation of `this` predicate.
    */
  override def toString: String = "true"
}

File: main/src/ca/uwaterloo/flix/runtime/solver/api/predicate/TruePredicate.java
package ca.uwaterloo.flix.runtime.solver.api.predicate;

/**
 * Represents the true predicate.
 */
public final class TruePredicate implements Predicate {

    /**
     * Returns a human-readable representation of `this` predicate.
     */
    @Override
    public String toString() {
        return "true.";
    }

}
